polkadot/runtime/rococo/constants/src/lib.rs

```diff
@@ -41,9 +41,9 @@ pub mod time {
 	use runtime_common::prod_or_fast;
 	pub const MILLISECS_PER_BLOCK: Moment = 6000;
 	pub const SLOT_DURATION: Moment = MILLISECS_PER_BLOCK;
-	pub const DEFAULT_EPOCH_DURATION: BlockNumber = prod_or_fast!(1 * HOURS, 1 * MINUTES);
+
 	frame_support::parameter_types! {
-		pub storage EpochDurationInBlocks: BlockNumber = DEFAULT_EPOCH_DURATION;
+		pub storage EpochDurationInBlocks: BlockNumber = prod_or_fast!(1 * HOURS, 1 * MINUTES, "ROCOCO_EPOCH_DURATION");
```

Comment:
```suggestion
		pub storage EpochDurationInBlocks: BlockNumber = option_env!("ROCOCO_EPOCH_DURATION").map(|s| s.parse().expect("`ROCOCO_EPOCH_DURATION` is not a valid `BlockNumber`")).unwrap_or(1 * MINUTES);
```

Let's take `1 Minute` as default.

Reply:
@bkchr one question: `prod_or_fast` is also used in other rococo parameters, e.g:
https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/rococo/src/lib.rs#L382-L387

If we read env directly, then we don't require to have `fast-runtime` feature enabled.

Question is: when changing epoch duration shall we also pass `fast-runtime` feature? Not sure if values of all those parameters shall be kept in sync somehow.

Reply:
I mean we may can change this hear and also make it dependent on the `fast-runtime` feature. My "reasoning" here more was that I would take 1 minute as the default value and there would exist no "faster default value". However, we could still make the env only work with the feature being enabled.

Reply:
Seems all `prod_or_fast` params in rococo runtime are now gone. So this is not a problem anymore.